api-scanner/src/test/java/net/corda/plugins/InternalAnnotationTest.java

```diff
@@ -0,0 +1,50 @@
+package net.corda.plugins;
+
+import org.gradle.testkit.runner.BuildResult;
+import org.gradle.testkit.runner.BuildTask;
+import org.gradle.testkit.runner.GradleRunner;
+import static org.gradle.testkit.runner.TaskOutcome.*;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.regex.Pattern;
+
+import static org.junit.Assert.*;
+
+public class InternalAnnotationTest {
+    @Rule
+    public final TemporaryFolder testProjectDir = new TemporaryFolder();
+
+    @Before
+    public void setup() throws IOException {
+        File buildFile = testProjectDir.newFile("build.gradle");
+        CopyUtils.copyResourceTo("internal-annotation/build.gradle", buildFile);
+    }
+
+    @Test
+    public void testInternalAnnotations() throws IOException {
+        BuildResult result = GradleRunner.create()
+            .withProjectDir(testProjectDir.getRoot())
+            .withArguments("scanApi", "--info")
+            .withPluginClasspath()
+            .build();
+        String output = result.getOutput();
+        System.out.println(output);
+
+        assertTrue(output.contains("net.corda.core.CordaInternal"));
```

Comment:
Can you use AssertJ instead? You can do this in one assert statement and it will helpfully print out the list if it fails.

Reply:
This is a (very) long `String` rather than a `List`. I am ensuring that the invisible annotation was at least mentioned in the `fast-classpath-scanner` output.

Reply:
OK, but my comment about AssertJ still stands with regards to a String